pt/lessons/libraries/guardian.md

```diff
@@ -207,13 +242,15 @@ end
 
 #### Login/Logout
 
-Fazer o login e logout de uma sessão do browser é muito simples. No controller de login:
+Fazer o login e logout de uma sessão do browser é muito simples. 
+No controller de login:
 
 ```elixir
 def create(conn, params) do
   case find_the_user_and_verify_them_from_params(params) do
     {:ok, user} ->
-      # Use access tokens. Other tokens can be used, like :refresh etc
+      # Use access tokens. 
+      #Other tokens can be used, like :refresh etc
```

Comment:
Minor typo: missing space

Reply:
Hi @anderkonzen! 
I used the [English](https://github.com/elixirschool/elixirschool/blob/master/en/lessons/libraries/guardian.md) text as a reference.